paddle/fluid/eager/auto_code_generator/eager_generator.cc

```diff
@@ -2481,7 +2481,7 @@ static std::string GenerateGradNodeHeaderContents(
       "%s\n"
       "    SetIsTensorWrappersCleared(true);\n"
       "  }\n"
-      "  std::string name() override { return \" GradNode%s \"; } \n "
```

Comment:
return of "name()" better in sync with the class name.

Reply:
Done